content/security/docs/pods.md

```diff
@@ -482,4 +482,3 @@ Policy-as-code and Pod Security Standards can be used to enforce this behavior.
 + A collection of common OPA and Kyverno [policies](https://github.com/aws/aws-eks-best-practices/tree/master/policies) for EKS.
 + [Policy based countermeasures: part 1](https://aws.amazon.com/blogs/containers/policy-based-countermeasures-for-kubernetes-part-1/)
 + [Policy based countermeasures: part 2](https://aws.amazon.com/blogs/containers/policy-based-countermeasures-for-kubernetes-part-2/)
-+ [Pod Security Policy Migrator](https://appvia.github.io/psp-migration/) a tool that converts PSPs to OPA/Gatekeeper, KubeWarden, or Kyverno policies 
```

Comment:
I'd like to keep this in.

Reply:
Restored. I do think the project's README could use some clarifications post-1.25, so I opened an issue in the project to that effect.